Carriers specs (#272)
PrestaShop/prestashop-specs: content/1.7/back-office/shipping/carriers/Add-new-edit-carrier.md

> ---

Reviewer: You can rename this file by:  add-edit-carrier.md instead of Add-new-edit-carrier.md

title: Add/edit carriet
weight: 1
---

 # General settings

 ## Call to action
 
A call to action button at the top right is displayed:

 - Cancel 

> Pressing the button redirects to [carriers-listing](./carriers-listing.md) page.

Reviewer: ```suggestion
Pressing the button redirects to the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page.
```

When an image is uploaded, it will appear in the logo field on the left, as well as in every panel of the Carrier Wizard.

 - Tracking URL. Has an explanation below the field: "For example: 'http://example.com/track.php?num=@' with '@' where the tracking number should appear.". 
A tooltip is displayed: "Delivery tracking URL: Type '@' where the tracking number should appear. It will be automatically replaced by the tracking number." The URL should be as displayed in the explanation below, if it's not entered by the explanation, and trying to save will show the message: "The url field is invalid.".

 ## Call to action
 
 - Previous. If any mandatory field was failed to enter, the button becomes non-pressable. If mandatory fields are correct, takes to the previous page. When in general settings, the button does not take anywhere.
 - Next. If any mandatory field was failed to enter, the button becomes non-pressable. If mandatory fields are correct, takes to he next page.

>  - Finish. Non pressable until all the pages mandatory fields are input correctly. When all mandatory fields are input correctly, takes to the [carriers-listing](./carriers-listing.md) page.

Reviewer: ```suggestion
 - Finish. Non pressable until all the pages mandatory fields are input correctly. When all mandatory fields are input correctly, takes to the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page.
```

# Multistore

This tab is shown when multistore is enabled.

The shop association block is displayed when the multistore is enabled and if several shops are configured.

> :point_right: See the [specs of the shop association block](../../../multistoregeneralspecs.md#shop-association-block)

Reviewer: ```suggestion
👉 See the [specs of the shop association block]({{<ref "../../multistoregeneralspecs.md#shop-association-block">}} "Shop association block") 

```

 ### Call to action
 
 - Previous. If any mandatory field was failed to enter, the button becomes non-pressable. If mandatory fields are correct, takes to the previous page. When in general settings, the button does not take anywhere.
 - Next. If any mandatory field was failed to enter, the button becomes non-pressable. If mandatory fields are correct, takes to he next page.

>  - Finish. Non pressable until all the pages mandatory fields are input correctly. When all mandatory fields are input correctly, takes to the [carriers-listing](./carriers-listing.md) page.

Reviewer: ```suggestion
 - Finish. Non pressable until all the pages mandatory fields are input correctly. When all mandatory fields are input correctly, takes to the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page.
```
